validator_manager/src/import_validator.rs

```diff
@@ -0,0 +1,550 @@
+pub const VC_TOKEN_FLAG: &str = "vc-token";
+pub const PASSWORD: &str = "password";
+pub const FEE_RECIPIENT: &str = "fee-recipient";
+pub const GAS_LIMIT: &str = "gas-limit";
+pub const BUILDER_PROPOSALS: &str = "builder-proposals";
+pub const BUILDER_BOOST_FACTOR: &str = "builder-boost-factor";
+pub const PREFER_BUILDER_PROPOSALS: &str = "prefer-builder-proposals";
+pub const ENABLED: &str = "enabled";
+
+
+pub const DETECTED_DUPLICATE_MESSAGE: &str = "Duplicate validator detected!";
+
+pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
+    App::new(CMD)
+        .about(
+            "Uploads standard keystore JSON format validator to a validator client using the HTTP API.",
+        )
+        .arg(
+            Arg::with_name(VALIDATOR_FILE_FLAG)
+                .long(VALIDATOR_FILE_FLAG)
+                .value_name("PATH_TO_JSON_FILE")
+                .help(
+                    "The path to a JSON file containing a validators to be \
+                    imported to the validator client.",
+                )
+                .required(true)
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name(VC_URL_FLAG)
+                .long(VC_URL_FLAG)
+                .value_name("HTTP_ADDRESS")
+                .help(
+                    "A HTTP(S) address of a validator client using the keymanager-API. \
+                    If this value is not supplied then a 'dry run' will be conducted where \
+                    no changes are made to the validator client.",
+                )
+                .default_value("http://localhost:5062")
+                .requires(VC_TOKEN_FLAG)
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name(VC_TOKEN_FLAG)
+                .long(VC_TOKEN_FLAG)
+                .value_name("PATH")
+                .help("The file containing a token required by the validator client.")
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name(IGNORE_DUPLICATES_FLAG)
+                .takes_value(false)
+                .long(IGNORE_DUPLICATES_FLAG)
+                .help(
+                    "If present, ignore any validators which already exist on the VC. \
+                    Without this flag, the process will terminate without making any changes. \
+                    This flag should be used with caution, whilst it does not directly cause \
+                    slashable conditions, it might be an indicator that something is amiss. \
+                    Users should also be careful to avoid submitting duplicate deposits for \
+                    validators that already exist on the VC.",
+                ),
+        )
+        .arg(
+            Arg::with_name(PASSWORD)
+                .long(PASSWORD)
+                .value_name("STRING")
+                .help("Password of keystore file.")
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name(FEE_RECIPIENT)
```

Comment:
If we keep all these optional flags, I would recommend to use the same name, for example, `suggested-fee-recipient` in the validator client:
https://github.com/sigp/lighthouse/blob/9e12c21f268c80a3f002ae0ca27477f9f512eb6f/validator_client/src/cli.rs#L161

Also recommend to update the help text, for example, can use the same help text from the validator client. The same goes for other flags like `builder-proposals`

```suggestion
            Arg::with_name(SUGGESTED_FEE_RECIPIENT)
```

Reply:
Updated in https://github.com/chong-he/lighthouse/commit/100ee224e52ade014de6373ed6eecf0d043e2746